.travis.yml

```diff
@@ -8,13 +12,24 @@ env:
 
 before_install:
   # Normal install of cargo-update
-  - if [[ ! -f ~/.cargo/bin/cargo-install-update ]]; then cargo install cargo-update; fi
+  - if [ ! -f ~/.cargo/bin/cargo-install-update ]; then
+      cargo install cargo-update;
+    fi
   # Conditional install/update of mdbook
   - cargo install-update -i mdbook
-  - export PATH=$HOME/.cargo/bin:$PATH
+  - export PATH="$HOME/.cargo/bin:$PATH"
 
 script:
   - mdbook build
 
-after_success:
-  - ./deploy.sh
+deploy:
+  - provider: pages
+    name: "Rust API Guidelines"
+    email: "guidelines@rust-lang.org"
+    skip-cleanup: true
+    local-dir: book
+    github-token: $GH_TOKEN
```

Comment:
This may failed due to incorrect token type: https://docs.travis-ci.com/user/deployment/pages/#setting-the-github-token.

Reply:
done